python/cuda_parallel/tests/test_reduce_api.py

```diff
@@ -2,9 +2,9 @@
 #
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-import numpy
+import cupy as cp
```

Comment:
suggestion: add cupy in https://github.com/NVIDIA/cccl/blob/89702de46541da0c920e619ccf14b96aa900aec8/python/cuda_parallel/setup.py#L115-L117

Reply:
btw to run CuPy we may need a part of CTK installed in the CI.

Reply:
added in commit a7a46d9